HTML4FUN

```diff
@@ -0,0 +1,146 @@
+      <li align="left">Капуцин</li>
+      </ul>
+    </li>
+    </ul>
+  </li>
+
+</ul>
+
+</body>
+</html></th>
+ <th width="65%" align="center" background="images/bek_564.jpg">
+    <table width="90%" border="2" cellspacing="1" bordercolor="#0000CC">
+       <caption>Така-собі табличка</caption>
+      <tr>
+       <td colspan="3" align="center"><input type="button" onclick="alert('Aloha! O_o ')" value="Hello!"></td>
+      </tr>
+      <tr>
+       <td align="center"> <input type="button" onclick="alert('^_^')" value="Konichiwa!"></td>
+       <td align="center"><input type="button" onclick="alert('(*_*)')" value="Guten Tag!"></td>
+       <td align="center"><input type="button" onclick="alert('$$$$$$$$$$$$$$$$!!!!!!!!')" value="You won 1m $!"></td>
+      </tr>
+      <tr>
+       <td align="center" colspan="2" rowspan="2"> <form action="Doc1.html">
+       <fieldset>
+       <legend>Особиста інформація:</legend>
+       Імя:<br>
+       <input type="text" name="firstname" value="Roy">
+       <br>
+       Прізвище:<br>
+       <input type="text" name="lastname" value="Killroy">
+        <br><br>
+        <input type="submit" value="Submit">
+       </fieldset>
+         </form>
+        </td>
+       <td align="center"><form action="Doc1.html">
+        <input type="checkbox" name="q1" value="a3">Я вмію літати
+       <br>
+       <input type="checkbox" name="q2" value="a4">Я вмію стрибати 
+       <br> <input type="checkbox" name="q5" value="a6">Я вмію плавати
+       <br>
+       <input type="checkbox" name="q7" value="q8">Я вмію бігати
+       <br>
+       <input type="submit">
+       </form> 
+       </td>
+      </tr>
+      <tr>
+       <td align="center">***</td>
+      </tr>
+      <tr>
+       <td align="center" colspan="2"> *** </td>
+       <td align="center"> *** </td>
+      </tr>
+      <tr>
+       <td align="center" colspan="3"> *** </td>
+      </tr>
+      <tr>
+       <td align="center">***</td>
+       <td align="center" rowspan="3" colspan="3"> *** </td>
+     </tr>
+     <tr>
+      <td align="center">***</td>
+     </tr>
+     <tr>
+      <td align="center">***</td>
+     </tr>
+     <tr>
+      <td align="center" colspan="3">***</td>
+     </tr>
+   </table>
+  <br>
+   <p style="color:red" align="center"><a href="https://www.google.com.ua/">Можливо, гугл тобі допоможе? :)</a>.</p>
+  <br>
+<p style="color:green" align="center"><a href="https://www.yandex.ua//"> Якщо справи ну дуже вже кепські:)</a>.</p>
+<br>
+  </th>
+  <th width="18%" align="center" bgcolor="#98F5FF"> <form action="Doc1.html">
+  Введіть будь-яке парне число:
+  <input type="number" name="points"
+   min="0" max="100" step="2" value="0">
+  <input type="submit">
+</form> </th>
+ </tr>
+ <tr>
+  <th  colspan=3 align="left" bgcolor="#1E90FF"><a href="http://www.w3schools.com">
+  <img src="http://www.w3schools.com/images/w3schools_green.jpg" alt="Посилання з картинки " style="width:270px;height:70px;border:0;">
```

Comment:
http://joxi.ru/DmBxQ1XIn1ZbAP

1. Resize images, so they look natural. 
2. Change them to Subaru topic.
3. Use static images inside repository instead of remote images via url.